exercises/practice/nucleotide-count/test/tests.chpl

```diff
@@ -0,0 +1,30 @@
+use UnitTest;
+
+use NucleotideCount;
+
+proc testEmptyStrand(test : borrowed Test) throws {
+  test.assertEqual(nucleotideCounts(""), [ "A" => 0, "C" => 0, "G" => 0, "T" => 0 ]);
+}
+
+proc testCanCountOneNucleotideInSingleCharacterInput(test : borrowed Test) throws {
+  test.assertEqual(nucleotideCounts("G"), [ "A" => 0, "C" => 0, "G" => 1, "T" => 0 ]);
+}
+
+proc testStrandWithRepeatedNucleotide(test : borrowed Test) throws {
+  test.assertEqual(nucleotideCounts("GGGGGGG"), [ "A" => 0, "C" => 0, "G" => 7, "T" => 0 ]);
+}
+
+proc testStrandWithMultipleNucleotides(test : borrowed Test) throws {
+  test.assertEqual(nucleotideCounts("AGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC"), [ "A" => 20, "C" => 12, "G" => 17, "T" => 21 ]);
+}
+
+proc testStrandWithInvalidNucleotides(test : borrowed Test) throws {
+  try {
```

Comment:
I think this is the older version of the template? A slightly better version

```chapel
proc testCaseUnderTest(test : borrowed Test) throws {
  try {
    steps(0); // Placeholder for function call here
    throw new AssertionError("Function was expected to throw an error");
  } catch e : IllegalArgumentError {
    test.assertEqual(e.message(), "Expected message here"); // placeholder for expected error message here
  } catch e {
    throw e;
  }
}
```

basically this rethrows the exception if it's not of the right kind.

Reply:
I must have copy/pasted the wrong one. Updated!